src/main/java/com/monkeyrizz/codeforest/controller/ProblemController.java

```diff
@@ -20,8 +21,16 @@ public ProblemController(ProblemService problemService) {
     }
 
     @GetMapping
-    @RequestMapping(path = "")
+    @RequestMapping(path = "list")
     public List<Problem> getProblems() {
         return problemService.getProblems();
     }
+
+    @GetMapping
+    @RequestMapping(path = "{id}")

```

Comment:
Ditto, please find a more intentional path name. We can pass in arguments to the controller using the `@RequestBody` annotation.

https://springframework.guru/requestbody-annotation/

Reply:
I will change this to problemId. Typically the HTTP GET request would not use a request body. It can have request body, but from what I've seen, people use {variable} to pass in args.